projects/batfish/src/org/batfish/grammar/cisco/Cisco_logging.g4

```diff
@@ -211,4 +214,24 @@ s_logging
       logging_common
       | logging_vrf
    )
-;
```

Comment:
Yes, it is about refactoring the rule fragments are used across multiple rules. For instance, there were varied uses of VRF clause, one with variable being captured, and other being ignored. This makes all uses uniform.